src/pages/signup-affiliates/components/_phone_number.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Header } from 'components/elements'
+import { localize } from 'components/localization'
+import OtpVerification from 'images/svg/signup-affiliates/code 2.svg'
```

Comment:
Let's rename this `code 2.svg` we shouldn't allow spaces on the filename

Reply:
okay